core/esmf-aspect-meta-model-java/src/main/java/org/eclipse/esmf/metamodel/impl/DefaultScalarValue.java

```diff
@@ -13,21 +13,29 @@
 
 package org.eclipse.esmf.metamodel.impl;
 
+import java.util.List;
 import java.util.Objects;
+import java.util.Set;
 import java.util.StringJoiner;
 
 import org.eclipse.esmf.aspectmodel.AspectModelFile;
+import org.eclipse.esmf.aspectmodel.loader.MetaModelBaseAttributes;
 import org.eclipse.esmf.aspectmodel.visitor.AspectVisitor;
 import org.eclipse.esmf.metamodel.Scalar;
 import org.eclipse.esmf.metamodel.ScalarValue;
+import org.eclipse.esmf.metamodel.datatype.LangString;
 
 public class DefaultScalarValue implements ScalarValue {
    private final Object value;
    private final Scalar type;
 
-   public DefaultScalarValue( final Object value, final Scalar type ) {
+   private final MetaModelBaseAttributes metaModelBaseAttributes;
+
+   public DefaultScalarValue( final Object value, final Scalar type,
+         final MetaModelBaseAttributes metaModelBaseAttributes ) {
```

Comment:
The baseAttributes should be the first constructor argument, like with all other Default implementation classes